src/Controller/Api/DatasetSubmissionController.php

```diff
@@ -376,6 +376,48 @@ public function getFile(DatasetSubmission $datasetSubmission, Request $request):
         }
     }
 
+    /**
+     * Returns a list of file IDs.
+     *
+     * @param DatasetSubmission $datasetSubmission The id of the dataset submission.
+     * @param Request           $request           The request object.
+     *
+     * @Route(
+     *     "/api/file_dataset_submission_ids/{id}",
+     *     name="pelagos_api_get_file_ids_dataset_submission",
+     *     methods={"GET"},
+     *     defaults={"_format"="json"},
+     *     requirements={"id"="\d+"}
+     *     )
+     *
+     * @View()
+     *
+     * @return Response
+     */
+    public function getFileIds(DatasetSubmission $datasetSubmission, Request $request): Response
```

Comment:
If this route is only use to delete files from a folder, then just call this function deleteFilesFromPath and actually delete them here. If it's in the same controller you may have to make the delete function a common function, but you can also call it from here. You can even all a routed controller from another controller.
It's called forwarding: https://symfony.com/doc/current/controller/forwarding.html

Reply:
We would be using this API, in the upcoming ticket for moving/renaming Folders

Reply:
Ok, we should keep this then, but maybe make a new function/route.